app/scripts/directives/resources.js

```diff
@@ -12,7 +81,27 @@ angular.module('openshiftConsole')
         // Optional URL for setting health checks on the resource when missing.
         addHealthCheckUrl: '@?'
       },
-      templateUrl: 'views/_pod-template.html'
+      templateUrl: 'views/_pod-template.html',
+      link: function(scope) {
+        scope.containers = _.get(scope.podTemplate, 'spec.containers', []);
+        scope.initContainers = _.get(scope.podTemplate, 'spec.initContainers', []);
+      }
+    };
+  })
+  .directive('podTemplateContainer', function() {
+    return {
+      restrict: 'E',
+      scope: {
+        podTemplateContainer: '=',
+        imagesByDockerReference: '=',
+        builds: '=',
+        detailed: '=?',
+        labelPrefix: '@?'
+      },
+      templateUrl: 'views/_pod-template-container.html',
+      link: function(scope) {
+        scope.container = scope.podTemplateContainer;
```

Comment:
Same comment here. We want the view to update when the podTemplateContainer value changes. I'd just use `podTemplateContainer` directly in the view. Or use

```js
scope: {
  container: '=podTemplateContainer',
  [...]
}
```

if you want the other name. I'd be OK just making the attribute `container` though